src/components/ConnectNode/modal.tsx

```
@@ -311,6 +319,26 @@ function ConnectNodeModal(props: ConnectNodeModalProps) {
+        dispatch(
+          nodeActionsAsync.getMinimumNetworkProbabilityThunk({
+            apiEndpoint,
+            apiToken: apiToken ? apiToken : '',
+          }),
+        );
+        dispatch(
+          nodeActionsAsync.getSessionsThunk({
+            apiEndpoint,
+            apiToken: apiToken ? apiToken : '',
+            protocol: 'udp'
+          }),
```

ESLint

[error] Missing trailing comma.

(comma-dangle)